src/main/java/frc/robot/commands/AutoScoreAndBalance.java

```diff
@@ -0,0 +1,26 @@
+package frc.robot.commands;
+
+import com.pathplanner.lib.PathConstraints;
+import com.pathplanner.lib.PathPlanner;
+import com.pathplanner.lib.PathPlannerTrajectory;
+
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.Calibrations;
+import frc.robot.subsystems.Lift;
+import frc.robot.subsystems.drive.DriveSubsystem;
+
+public class AutoScoreAndBalance extends SequentialCommandGroup {
+    private PathPlannerTrajectory traj = PathPlanner.loadPath("Trajectory",
```

Comment:
Make "Trajectory" a placeholder string variable, since we don't know what specific trajectory it is.